docs/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - unified logging across the whole OSIDB
+- validate differences between Red Hat and NVD CVSS score and severity (OSIDB-333)
```

Comment:
Good point :smile: I would actually be for providing those Jira IDs when the change is based on a Jira issue. It is true that for any external contributor/watcher it has no meaning (also no harm I think) but we still have majority of our focus internally, we share our CHANGELOG there with every release, and this may clarify the changes etc. WDYT?

Reply:
I agree with @osoukup I think it's a good idea to have it (we can do it retroactively at some point), it could help internal stakeholders to follow the CHANGELOG better